WordPress/src/main/java/org/wordpress/android/ui/jetpack/backup/progress/BackupDownloadProgressViewModel.kt

```diff
@@ -0,0 +1,15 @@
+package org.wordpress.android.ui.jetpack.backup.progress
+
+import androidx.lifecycle.ViewModel
+import javax.inject.Inject
+
+class BackupDownloadProgressViewModel @Inject constructor() : ViewModel() {
+    private var isStarted = false
+
+    fun start() {
+        if (isStarted) {
```

Comment:
Minor (🔍 ): Consider making this a one-liner. This way `if (started) return` and `isStarted = true` will be as close to each other as possible.

Reply:
This is another one of those instances in which the code base does both. 🤷

Reply:
Definitely, I agree on that, in my case, if I were to write this code I would always choose the one-liner alternative for such simple logic. This is also what the official Kotlin style guide suggests, thus it makes it easier on me deciding on it.